llama/grammar_test.go

```diff
@@ -0,0 +1,80 @@
+package llama
+
+import (
+	"bufio"
+	"bytes"
+	"strings"
+	"testing"
+)
+
+// https://github.com/ollama/ollama/issues/7978
+const issue7978JSONSchema = `{
```

Comment:
```suggestion
const OrderedJSONSchema = `{
```
(or something along those lines)

Reply:
See note above. This is specific to the issue. Keeping it as-is.